app/javascript/src/components/Dashboard/Notes/MenuBar.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+import { Search, Plus, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar as NeetoUIMenuBar } from "neetoui/layouts";
+
+import { TAGS, TYPES, SEGMENTS } from "./constants";
+
+const MenuBar = ({ showMenu }) => (
+  <NeetoUIMenuBar showMenu={showMenu} title="Notes">
+    {TYPES.map((type, idx) => (
+      <NeetoUIMenuBar.Block key={idx} label={type.label} count={type.count} />
+    ))}
+
+    <NeetoUIMenuBar.Block label={"All"} count={10} />
```

Comment:
Please don't add curly braces for strings. It's not redundant. Refer: https://www.bigbinary.com/react-best-practices/miscellaneous-tips#rendering-strings-and-blank-spaces-in-jsx